ONSdigital/eq-author-app
# Move husky to root

### What is the context of this PR?
Stop maintaining different huskys which are all trying to edit the one
`.git` folder. So instead just set this up at the root.

Closes #106 

### How to review 
1. Run yarn at root
2. Run yarn in each project
3. Ensure you can commit successfully from each project
4. Ensure your editor still runs prettier as expected.
